include/boost/config/compiler/visualc.hpp

```diff
@@ -218,6 +218,10 @@
 //
 #define BOOST_NO_TWO_PHASE_NAME_LOOKUP
 #define BOOST_NO_CXX11_SFINAE_EXPR
+// C++ 17:
+#define BOOST_NO_CXX17_STRUCTURED_BINDINGS
+#define BOOST_NO_CXX17_INLINE_VARIABLES
+#define BOOST_NO_CXX17_FOLD_EXPRESSIONS
```

Comment:
Is there a reason for unconditionally defining these here, rather than using the std feature macros?  Just curious, John.